packages/api-client/src/api/getCategory/defaultQuery.ts

```diff
@@ -0,0 +1,61 @@
+import gql from 'graphql-tag';
+
+export default gql`
+  query collections {
+	collections{
+      totalItems
+      items {
+        id
+        createdAt
+        updatedAt
+        languageCode
+        name
+        breadcrumbs {
+          id
+          name
+          slug
+        }
+        description
+        position
+        slug
+        featuredAsset {
+          id
+          name
+        }
+        assets {
+          id
+          name
+        }
+        parent {
+          id
+          name
+        }
+        children {
+          id
+          name
+        }
+        filters {
+          code
+        }
+        translations {
```

Comment:
Normally you don't need to fetch the `translations` field, as the correct translated values (name, slug, description) are automatically populated into the Collection type at runtime by Vendure, based on the current context (i.e. the current channel's default language, or the current language as specified by the [languageCode query param](https://www.vendure.io/docs/storefront/shop-api-guide/#universal-parameters).